backslash/contrib/slash_plugin.py

```diff
@@ -170,6 +170,32 @@ def test_start(self):
     def test_skip(self, reason=None):
         self.current_test.mark_skipped(reason=reason)
 
+    @slash.plugins.registers_on(None)
+    def is_session_exist(self, session_id):
+        try:
+            self.client.api.get('/rest/sessions/{0}'.format(session_id))
+            return True
+        except HTTPError:
+            return False
+
+    @handle_exceptions
+    @slash.plugins.registers_on(None)
+    def get_tests_to_resume(self, session_id):
+        tests_to_resume = []
+        has_more = True
+        page_id = 1
+        while has_more:
+            params = {'session_id':session_id, 'page':page_id, 'show_successful':'false', 'page_size':'200'}
+            response = self.client.api.get('/rest/tests', params=params)
+            has_more = response['meta']['has_more']
+            page_id += 1
+            for test_item in response['tests']:
+                test_name = '{0}:{1}'.format(test_item['info']['file_name'], test_item['info']['name'])
+                if test_item['variation']:
+                    test_name = '{0}{1}'.format(test_name, test_item['variation'])
```

Comment:
I don't think the formatting should be done here. This API should return rich structures about what should be run. If necessary, formatting can be done in the client code